### DISCUSSION TITLE
bg1.png (Background)

### DISCUSSION THREAD

**Discussion by billsecond** [Leantime/leantime#720] [General] [answered]
First, I love the tool, its amazing!  
I am also a developer and wanted to know if I can have a copy of bg1.png to do what ever I want with it license free?

Let me know!  Thanks!

**Comment by marcelfolaron** [comment, marked as answer]
It's part of this open source project and thus the "GNU General Public License v2.0" applies: https://github.com/Leantime/leantime/blob/master/LICENSE
